Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2_test.py

```diff
@@ -1725,6 +1730,68 @@ def test_test_module(mocker):
     with pytest.raises(Exception) as e:
         module(client)
     assert "Test button cannot be used when using OAuth 2.0" in str(e)
+    
+def test_invalid_private_key():
+    """
+    Given:
+    - Invalid format of private key
+    When:
+    - creating the JWT
+    Then:
+    - Raise a Value error with informative message
+    """
+    params = {
+    'private_key': '-----INVALID FORMAT----- test_token -----INVALID FORMAT-----',
+    'kid': 'test1',
+    'sub': 'test'
+}
+
+    with pytest.raises(ValueError) as e:
+        Client('server_url', 'sc_server_url', 'cr_server_url', 'username', 'password', 'verify', 'fetch_time',
+            'sysparm_query', sysparm_limit=10, timestamp_field='opened_at', ticket_type='incident',
+            get_attachments=False, incident_name='description', oauth_params=OAUTH_PARAMS, jwt_params = params)
+    assert "Invalid private key format" in str(e)
+
+    
+    
+def test_jwt_checker(mocker):
+    """
+    Given:
+    - private key
+    When:
+    - creating a jwt
+    Then:
+    - (a) that the return type is a string
+    - (b) validate the pem format
+    """
+
+    mocker.patch.object(jwt, "encode", return_value="")
+    client = Client('server_url', 'sc_server_url', 'cr_server_url', 'username', 'password', 'verify', 'fetch_time',
+                    'sysparm_query', sysparm_limit=10, timestamp_field='opened_at', ticket_type='incident',
+                    get_attachments=False, incident_name='description', oauth_params=OAUTH_PARAMS, jwt_params = JWT_PARAMS)
+    test_token = client.check_private_key(JWT_PARAMS['private_key'])
+    assert isinstance(test_token, str)
+    assert test_token.startswith('-----BEGIN PRIVATE KEY-----')
+    assert test_token.endswith('-----END PRIVATE KEY-----')
```

Comment:
Lets check that the test_token does not contain unwanted chars (spaces for example)